lib/operations/logic/strictEqual.js

```diff
@@ -0,0 +1,6 @@
+function strictEqual(a, b) {
+  return a === b;
+}
+
+strictEqual.code = '===';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`===` is an existing operator

Reply:
yes 👍